### PR TITLE
proxyless: rename proxyless option to nativeAutomation

### PR DESCRIPTION
**Pull request by Artem-Babich**
## Purpose
_Describe the problem you want to address or the feature you want to implement._

## Approach
_Describe how your changes address the issue or implement the desired functionality in as much detail as possible._

## References
_Provide a link to the existing issue(s), if any._

## Pre-Merge TODO
- [ ] Write tests for your proposed changes
- [ ] Make sure that existing tests do not fail

### CODE REVIEW

File: test/functional/fixtures/browser-provider/browser-reconnect/run-log-error-on-disconnect-test.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ createTestCafe()
                 return testName === 'Should log error on browser disconnect';
             })
             .reporter(shouldAttachReporter ? reporter : [])
-            .run({ proxyless: config.proxyless });
+            .run({ nativeAutomation: config.proxyless });
```
Comment (miherlosev) [inline test/functional/fixtures/browser-provider/browser-reconnect/run-log-error-on-disconnect-test.js:27]: Rename the `config.proxyless` property and related test stuff.

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: gulp/helpers/test-functional.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ module.exports = async function testFunctional (src, testingEnvironmentName, { e
         process.env.EXPERIMENTAL_DEBUG = 'true';
 
     if (isProxyless)
```
Comment (miherlosev) [inline gulp/helpers/test-functional.js:37]: Rename to `nativeAutomation` here too.

File: test/functional/fixtures/api/es-next/iframe-switching/test.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 const { expect }                     = require('chai');
-const { errorInEachBrowserContains } = require('../../../../assertion-helper.js');
-const { skipDescribeInProxyless }    = require('../../../../utils/skip-in');
+const { errorInEachBrowserContains }     = require('../../../../assertion-helper.js');
```
Comment (miherlosev) [inline test/functional/fixtures/api/es-next/iframe-switching/test.js:2]: Fix align of the `=` symbols.

File: test/functional/fixtures/api/es-next/upload/test.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
 const { expect }          = require('chai');
-const path                = require('path');
-const { skipInProxyless } = require('../../../../utils/skip-in');
+const path                       = require('path');
```
Comment (miherlosev) [inline test/functional/fixtures/api/es-next/upload/test.js:2]: Fix align of the `=` symbols.

File: test/functional/fixtures/api/json/test.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 const { expect }          = require('chai');
-const path                = require('path');
-const { skipInProxyless } = require('../../../utils/skip-in');
+const path                       = require('path');
+const { skipInNativeAutomation } = require('../../../utils/skip-in');
```
Comment (miherlosev) [inline test/functional/fixtures/api/json/test.js:3]: Fix align of the `=` symbols.

File: test/functional/fixtures/concurrency/test.js [hunk 6]
```diff
@@ -3,8 +3,8 @@ const { expect }          = require('chai');
 const isCI                = require('is-ci');
 const config              = require('../../config');
 const { createReporter }  = require('../../utils/reporter');
-const testInfo            = require('./test-info');
-const { skipInProxyless } = require('../../utils/skip-in');
+const testInfo                   = require('./test-info');
+const { skipInNativeAutomation } = require('../../utils/skip-in');
```
Comment (miherlosev) [inline test/functional/fixtures/concurrency/test.js:7]: Fix align of the `=` symbols.

File: test/functional/fixtures/proxy/test.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 const os                  = require('os');
 const http                = require('http');
-const { expect }          = require('chai');
-const { skipInProxyless } = require('../../utils/skip-in');
+const { expect }                 = require('chai');
```
Comment (miherlosev) [inline test/functional/fixtures/proxy/test.js:3]: Fix align of the `=` symbols.

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: src/errors/runtime/templates.js [hunk 8]
```diff
@@ -144,5 +144,5 @@ export default {
     [RUNTIME_ERRORS.invalidCustomActionType]:                        `TestCafe cannot parse the "{actionName}" action, because the action definition is invalid. Format the definition in accordance with the custom actions guide: ${ DOCUMENTATION_LINKS.CUSTOM_ACTIONS }`,
     [RUNTIME_ERRORS.cannotImportESMInCommonsJS]:                     'Cannot import the {esModule} ECMAScript module from {targetFile}. Use a dynamic import() statement or enable the --esm CLI flag.',
     [RUNTIME_ERRORS.proxyInitializedMoreThanOnce]:                   'The proxy was initialized more than once.',
-    [RUNTIME_ERRORS.setProxylessForUnsupportedBrowsers]:             'The "proxyless" mode is not supported in the {browsers} browser{ending}.',
+    [RUNTIME_ERRORS.setProxylessForUnsupportedBrowsers]:             'The following browser(s) do not support the Native Automation mode: {browsers}.',
```
[the remark this answers is not in the document]
Comment (Artem-Babich) [reply src/errors/runtime/templates.js:147]: yes

File: src/cli/argument-parser/index.ts [hunk 9]
```diff
@@ -167,7 +167,7 @@ export default class CLIArgumentParser {
             .option('--test-meta <key=value[,key2=value2,...]>', 'filter tests by metadata')
             .option('--fixture-meta <key=value[,key2=value2,...]>', 'filter fixtures by metadata')
             .option('--debug-on-fail', 'pause tests on failure')
-            .option('--proxyless', 'enable proxyless mode')
+            .option('--native-automation', 'enable native automation')
```
[the remark this answers is not in the document]
Comment (Artem-Babich) [reply src/cli/argument-parser/index.ts:170]: yes

**Review by miherlosev — APPROVED** [review]

**Review by aleks-pro — APPROVED** [review]

**Pull request merged**